fitbenchmarking/fitbenchmarking
# Move jacobian creation out of loop

#### Description of Work

Fixes #707 


#### Testing Instructions

1. Check change hasn't broken anything

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
